Resources/Locale/en-US/pai/pai-system.ftl

```diff
@@ -8,10 +8,10 @@ pai-system-role-description = Seja o amigo eletrônico de alguém!
                               (Memórias *não* inclusas.)
 
 pai-system-wipe-device-verb-text = Remover pAI
-pai-system-wiped-device = A IAp for removida do dispositivo.
+pai-system-wiped-device = A pAI for removida do dispositivo.
```

Comment:
Aqui deveria ser "foi" ao invés de "for".

Reply:
você acha melhor deixar pAI ou IAp?

Reply:
Cara, pior que n sei. Deixa pAI mesmo, e troca o for por foi.